src/interactive/AdminBar.jsx

```diff
@@ -0,0 +1,239 @@
+// @flow
+import React from 'react';
+import cx from 'classnames';
+
+import Button from '../forms/Button';
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import Icon from '../media/Icon';
+import Section from '../layout/Section';
+import Tooltip from './Tooltip';
+import SelectInput from '../forms/SelectInput';
+
+type DropdownProps = {
+	group: Group,
+	event?: EventInfo,
+	host: string,
+};
+
+type Props = {
+	group: Group,
+	event?: EventInfo,
+	isQL: boolean,
+	isAdmin: boolean,
+	isProd: boolean,
+	self: Self,
+	nodeEnv: string,
+};
+
+type State = {
+	highlightValue: string,
+	showHighlighter: boolean,
+};
+
+const DropdownContent = ({ group, event, host }: DropdownProps): React$Node => (
+	<ul className="padding--all">
+		{group.organizer && (
+			<li>
+				<a
+					href={`https://www.${host}/ql/?qlMemberId=${group.organizer.id}`}
+					className="link"
+					target="_blank"
+					rel="noopener noreferrer"
+				>
+					QL Org
+				</a>
+			</li>
+		)}
+		<li>
+			<a
+				href={`https://admin.${host}/admin/group.jsp?g=${group.id}&op=search`}
+				className="link"
+				target="_blank"
+				rel="noopener noreferrer"
+			>
+				Group
+			</a>
+		</li>
+		{group.organizer && (
+			<li>
+				<a
+					href={`https://admin.${host}/admin/member.jsp?m=${
+						group.organizer.id
+					}`}
+					className="link"
+					target="_blank"
+					rel="noopener noreferrer"
+				>
+					Org
+				</a>
+			</li>
+		)}
+		{event !== undefined &&
```

Comment:
i think this can be just `event &&` since `undefined` would evaluate as falsy

Reply:
Good point, I took from mup-web, but you are right that it doesn't need to be so verbose.